### PR TITLE
Minus stun time fix

### PR DESCRIPTION
**Pull request by Marvizm**

### CODE REVIEW

**Comment by howardchung** [comment]
I'd prefer to just show the actual data.

**Comment by Marvizm** [comment]
It's not first time when I saw an issue with negative stun value, I can't fix it in replay, but I can fix it here.

**Comment by howardchung** [comment]
Yeah, but then we're hiding the actual data, even if we suspect it's wrong.

**Comment by Marvizm** [comment]
You can stun on 4 sec or on 0 sec but not on -0.93 in dota 2. No need to hide data when it's easy to fix

**Comment by howardchung** [comment]
My point is that we're obscuring the actual value in the replay, even if it's wrong.

**Comment by Marvizm** [comment]
but it is stupid, if there is an error, it needs to be fixed

**Comment by howardchung** [comment]
I don't know if it's an error.  I suspect it's a replay bug, but I have no way of knowing for sure.

@albertcui, @nicholashh ?

File: views/match/match_performances.jade [hunk 1]
```diff
@@ -23,7 +23,7 @@ block match_content
             //td.activate(data-index=i, data-type="sen") #{player.sen_log.length || "-"}
             td.rankable #{(Number(player.lane_efficiency)*100).toFixed(2)}
             td.rankable #{player.lh_t && player.lh_t[10] ? player.lh_t[10] : "-"}
-            td.rankable #{Number(player.stuns).toFixed(2)}
+            td.rankable #{Number(player.stuns > 0 ? player.stuns : player.stuns == 0 ? 0 : 0).toFixed(2)}
```
Comment (howardchung) [inline views/match/match_performances.jade]: Should probably just Math.max(player.stuns, 0)

**Comment by howardchung** [comment]
I suppose this is ok beause those interested in raw data can use the API.

@albertcui review

**Pull request merged**